custom_image_utils/args_inferer.py

```diff
@@ -165,9 +165,13 @@ def _get_dataproc_image_path_by_version(version):
     if stdout:
       # in case there are multiple images
       parsed_lines = stdout.decode('utf-8').strip().split('\n')
+      expected_prefix = "dataproc-{}-{}".format(parsed_version[0], parsed_version[1])
       for line in parsed_lines:
         parsed_image = line.split(",")
         if len(parsed_image) == 2:
+          if not parsed_image.startswith(expected_prefix):
+            # Not a regular dataproc release image. Maybe a custom image with same label.
```

Comment:
Add a log for it.

Reply:
Can you also add a check that if more than one valid images are found, fail the build with an error.

Reply:
Done.